benchexec/tools/mopsa.py

```diff
@@ -0,0 +1,55 @@
+# This file is part of BenchExec, a framework for reliable benchmarking:
+# https://github.com/sosy-lab/benchexec
+#
+# SPDX-FileCopyrightText: 2022 Raphaël Monat
+#
+# SPDX-License-Identifier: Apache-2.0
+
+import benchexec.tools.template
+import benchexec.result as result
+
+import re, json, subprocess
```

Comment:
Please keep imports on separate lines. `re` is unused, btw.

Reply:
Cleaned. I also applied `black` as requested.